scripts/start.sh

```diff
@@ -292,7 +292,15 @@ fi
 if [ "${BACKUP_ENABLED}" = true ]; then
     echo "BACKUP_ENABLED=${BACKUP_ENABLED}"
     
-    echo "$BACKUP_CRON_EXPRESSION bash /usr/local/bin/backup" > "/home/steam/server/crontab"
+    echo "$BACKUP_CRON_EXPRESSION bash /usr/local/bin/backup" >> "/home/steam/server/crontab"
+fi
+
+if [ "${UPDATE_ENABLED}" = true ]; then
+    echo "UPDATE_ENABLED=${UPDATE_ENABLED}"
+    echo "$UPDATE_CRON_EXPRESSION bash /usr/local/bin/update" >> "/home/steam/server/crontab"
+fi
+
+if [ "${UPDATE_ENABLED}" = true ] || [ "${BACKUP_ENABLED}" = true ]; then
```

Comment:
Since `UPDATE_ON_BOOT` I would also check that before making the cronjob.
```suggestion
if [ "${UPDATE_ENABLED}" = true ] && [ "${UPDATE_ON_BOOT}" = "true" ]; then
    echo "UPDATE_ENABLED=${UPDATE_ENABLED}"
    echo "$UPDATE_CRON_EXPRESSION bash /usr/local/bin/update" >> "/home/steam/server/crontab"
fi

if ([ "${UPDATE_ENABLED}" = true ] && [ "${UPDATE_ON_BOOT}" = "true" ]) || "${BACKUP_ENABLED}" = true ]; then
```